examples/server/main.rs

```diff
@@ -95,85 +108,65 @@ fn handle_client(mut stream: TcpStream) -> Result<()> {
         );
 
         // complete the opening handshake with the client
-        framer.accept(&mut stream, &websocket_context)?;
+        framer.accept(stream, &websocket_context)?;
         println!("Websocket connection opened");
 
         // read websocket frames
-        while let ReadResult::Text(text) = framer.read(&mut stream, &mut frame_buf)? {
+        while let ReadResult::Text(text) = framer.read(stream, &mut frame_buf)? {
             println!("Received: {}", text);
 
             // send the text back to the client
             framer.write(
-                &mut stream,
+                stream,
                 WebSocketSendMessageType::Text,
                 true,
-                text.as_bytes(),
+                format!("shut the fuck up {}", text).as_bytes(),
```

Comment:
What's this?

Reply:
oh wait

Reply:
just uhhhhhhhhh let's say it never happened